jhove-core/src/main/java/edu/harvard/hul/ois/jhove/NisoImageMetadata.java

```diff
@@ -20,6 +20,11 @@
 	/** Logger for this class. */
     private static final Logger LOGGER = Logger.getLogger(NisoImageMetadata.class.getCanonicalName());
 
+    /** private String constants. */
+    private static final String YES = "Yes";
+    private static final String NO = "No";
+    private static final String COLON = ":";

```

Comment:
The definition of the constant `COLON` is a classic "matter of taste" and lacks a definitive answer in lieu of a style guide. Over the years I've taken the definitive stance of wavering on this. This [Stack Exchange question](https://stackoverflow.com/questions/33183525/does-java-define-constants-for-any-characters-such-as-space) states the case against. If it's to be a constant, I'd adopt a name that indicates its use, in this case, `TIME_SEP` perhaps?

Reply:
OK, I got the one for the Niso Image Metadata. I'll update it, even if I get personal sophomoric enjoyment from using COLON as a constant. :) Not sure if I understand what is the issue/if there is an issue with ConfigHandler.java.